nova/nova/models.py

```diff
@@ -235,3 +229,29 @@ class Order(models.Model):
     choice = models.SmallIntegerField(choices=ORDER_CHOICES)
 
     tr_eq = models.ForeignKey(TradingEquipment, on_delete=models.CASCADE)
+
+
+class Notification(models.Model):
+    REASON_CHOICES = (
+        ('comment_create', 'comment_create'),
+        ('article_create', 'article_create'),
+        ('annotation_create', 'annotation_create')
+    )
+
+    SOURCE_TYPE_CHOICES = (
+        ('following_user', 'following_user'),
```

Comment:
Trading equipment and portfolio can also be added here but if you think that it can be done in another PR it is alright.